week2/Program.cs

```diff
@@ -0,0 +1,183 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+// Homework
+app.MapGet("/hw-task", (string str) =>
+{
+    string[] words = str.Split(' ');
+
+    // method 1
+    List<Tuple<string, int>> result = new List<Tuple<string, int>>();
+    foreach (var item in words)
+    {
+        result.Add(new Tuple<string, int>(item, item.Length));

```

Comment:
This will not count the frequency of the word in the sentence but instead, it would measure the length of the word.